elk/tests/utils_evaluation/test_eval_parser.py

```diff
@@ -0,0 +1,25 @@
+import json
+from elk.utils_evaluation.parser import get_parser
+
+
+def test_args_no_lowercase():
```

Comment:
hah, this seems a little overkill but sure why not

Reply:
Wait but shouldn't this be named `test_args_no_underscores`?

Reply:
> hah, this seems a little overkill but sure why not

Haha, perhaps. If the number of parameters begins to expect over time then such tests may be useful.